addons/website_event_track/models/event_track.py

```diff
@@ -596,7 +594,7 @@ def _get_track_suggestions(self, restrict_domain=None, limit=None):
 
         track_candidates = track_candidates.sorted(
             lambda track:
-                (track.is_published,
+                (track.is_visible,
```

Comment:
Oh second use of is_visible...  Would instead keep sorting on is published. Anyway tracks that are not visible in agenda should not be displayed here, am I wrong ? So sorting on "is_visible" is the same as sorting on "website_published", unless I am mistaken.